src/content/docs/dropins/user-auth/auth-functions.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: User Auth Functions
+description: Learn about the API functions provided by the Product Details dropin.
+---
+
+
+## confirmEmail
+
+The `confirmEmail` function uses the supplied `customerEmail` and `customerConfirmationKey` parameters to complete the customer activation process. Adobe Commerce sends the confirmation key to the customer when they request to create an account. The function calls the [`confirmEmail` mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/confirm-email/).
+
+confirmEmail({
+  customerEmail: "abc@example.com",
+  customerConfirmationKey: "1234567890",
+});
+
+### Syntax
```

Comment:
Style issue not related to this PR, but if you don't separate headings from code blocks with a paragraph/lead-in sentence it renders without enough space between and looks bad:

![Screenshot 2024-09-19 at 8 32 55 AM](https://github.com/user-attachments/assets/cc3c1c09-c6b0-45f5-bf75-73ef05a04d72)

We should either add a boilerplate lead-in sentence or open a ticket in our backlog to fix the style to increase spacing.